fe/fe-core/src/main/java/org/apache/doris/planner/StreamLoadPlanner.java

```diff
@@ -203,7 +203,7 @@ public TExecPlanFragmentParams plan(TUniqueId loadId) throws UserException {
     }
 
     // get all specified partition ids.
-    // if no partition specified, return all partitions
+    // if no partition specified, return null
     private List<Long> getAllPartitionIds() throws DdlException, AnalysisException {
```

Comment:
You change it to return null, but don' t change the caller to handle null? Such as in `StreamLoadPlanner`?

Reply:
I unified handle this in the OlapTableSink.